test_tableslurp.sh

```diff
@@ -71,10 +71,10 @@ BACKUP_BUCKET="${BUCKET}"
 DEST_S3_UPDOWN_FILE=s3://"${BUCKET}"/"${CONTAINER}":"${SRC_LOCAL_UPDOWN_FILE}"
 DEST_S3_INDEX_FILE=s3://"${BUCKET}"/"${CONTAINER}":/"${INDEX_FILE}"
 
-echo "Source on local for file to upload:" "${SRC_LOCAL_UPDOWN_FILE}"
-echo "Dest on S3 for file to upload:\t" "${DEST_S3_UPDOWN_FILE}"
-echo "Source on local for index file to upload:" "${SRC_LOCAL_INDEX_FILE}"
-echo "Dest on S3 for index file to upload:\t" "${DEST_S3_INDEX_FILE}"
+printf "%s\n Source on local for file to upload:" "${SRC_LOCAL_UPDOWN_FILE}"
+printf "%s\n Dest on S3 for file to upload:\t" "${DEST_S3_UPDOWN_FILE}"
+printf "%s\n Source on local for index file to upload:" "${SRC_LOCAL_INDEX_FILE}"
+printf "%s\n Dest on S3 for index file to upload:\t" "${DEST_S3_INDEX_FILE}"
```

Comment:
I think you meant to put the `%s \n` at the end of the first strings, not the beginning. For example

```
printf "Source on local for file to upload: %s\n" "${SRC_LOCAL_UPDOWN_FILE}"
```

instead of

```
printf "%s\n Source on local for file to upload:" "${SRC_LOCAL_UPDOWN_FILE}"
```

Right now you're getting funny output that looks like this:

```
/tmp/tablesnap-test/aws_s3_cp.txt
 SRC_LOCAL_UPDOWN_FILE:		/tmp/tablesnap-test/orig_test_file.txt
 SRC_LOCAL_ORIG_TEST_FILE:
```

Reply:
I missed seeing that in the output. Thanks.

Reply:
Cpnfirmed fixed.